conda-recipe/conda_build_config.yaml

```diff
@@ -1,7 +1,7 @@
 #For some reason the resolver decides to use an old version of numpy
 #without this
 numpy:
-    - 1.19
+    - 1.21
```

Comment:
Is this still necessary?

Reply:
I don't know whether it is necessary but I'm not touching conda with a stick in the next three months.